File: src/main/java/no/nav/domain/pensjon/regler/vedtak/TidligereGjenlevendePensjon.java
package no.nav.domain.pensjon.regler.vedtak;

/**
 * @author Steinar Hjellvik (Decisive) - PK-11391
 */
public class TidligereGjenlevendePensjon extends AbstraktBeregningsvilkar {

    /**
     * Angir om bruker mottok GJP som følge av avdødes dødsfall.
     */
    private boolean sokerMottokGJPForAvdod;

    /**
     * Angir om avdøde hadde inntekt på minst 1G før dødsfall
     */
    private boolean arligPGIMinst1G;


}

File: src/main/kotlin/no/nav/pensjon/regler/domain/vedtak/TidligereGjenlevendePensjon.kt
package no.nav.pensjon.regler.domain.vedtak

class TidligereGjenlevendePensjon : AbstraktBeregningsvilkar() {
    /**
     * Angir om bruker mottok GJP som fålge av avdødes dødsfall.
     */
    var sokerMottokGJPForAvdod = false

    /**
     * Angir om avdøde hadde inntekt på minst 1G før dødsfall
     */
    var arligPGIMinst1G = false

}